standard_init_linux.go

```diff
@@ -77,6 +78,9 @@ func (l *linuxStandardInit) Init() error {
 	if err != nil {
 		return err
 	}
+	if err := seccomp.InitSeccomp(l.config.Config.SeccompConfig); err != nil {
```

Comment:
Children of container init will inherit automatically. Processes joined to the namespace (via exec, for example) will not inherit, so I have those execute seccomp_init() again to acquire the restrictions before joining the namespace.